tests/lsdb/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import hipscat as hc
+import lsdb
+import pytest
+from hipscat.io.file_io import file_io
+
+SMALL_SKY_DIR_NAME = "small_sky"
+SMALL_SKY_XMATCH_NAME = "small_sky_xmatch"
+SMALL_SKY_ORDER1_DIR_NAME = "small_sky_order1"
+XMATCH_CORRECT_FILE = "xmatch_correct.csv"
+XMATCH_CORRECT_005_FILE = "xmatch_correct_0_005.csv"
+XMATCH_MOCK_FILE = "xmatch_mock.csv"
+
+
+@pytest.fixture
+def test_data_dir_cloud(example_cloud_path):
+    return os.path.join(example_cloud_path, "lsdb", "data")
+
+
+@pytest.fixture
+def small_sky_dir_cloud(test_data_dir_cloud):
+    return os.path.join(test_data_dir_cloud, SMALL_SKY_DIR_NAME)
```

Comment:
I see a lot of these are duplicated between the hipscat and lsdb tests, do you think it's good to keep them seperate over putting them in a higher level shared contest?

Reply:
The full paths are different, since the `test_data_dir_cloud` is different between lsdb and hipscat.